src/map/lua/lua_zone.h

```diff
@@ -44,6 +46,9 @@ class CLuaZone
     void setLocalVar(const char* key, uint32 value);
     void resetLocalVars();
 
+    auto createSharedPool() -> std::optional<CLuaTreasurePool>;
```

Comment:
How will these be intended to be used? On zone setup? Like when a group enters dynamis, etc?